app/javascript/gobierto_visualizations/webapp/containers/contracts/Summary.vue

```diff
@@ -269,7 +273,7 @@ export default {
     goesToItem(event) {
       const { id } = event
       // eslint-disable-next-line no-unused-vars
-      this.$router.push(`adjudicaciones/${id}`).catch(err => {})
+      this.$router.push(`/visualizaciones/contratos/adjudicaciones/${id}`).catch(err => {})
```

Comment:
Try always to parametrize route paths as it's done in some other modules- This literal string should be only present at the router configuration.

You can create a new issue for refactor all this, since there's no route parametrized yet